### PR TITLE
Add shu-mutou to sig-ui leads

### PR DESCRIPTION
**Pull request by shu-mutou**
Contexts:
* https://groups.google.com/g/kubernetes-dev/c/_sGC7W4YiVc/m/QkePzg--AgAJ
* #2781

### CODE REVIEW

File: config/kubernetes/org.yaml [hunk 1]
```diff
@@ -1389,6 +1389,7 @@ teams:
     members:
     - floreks
     - maciaszczykm
+    - shu-mutou
```
[the remark this answers is not in the document]
Comment (maciaszczykm) [reply config/kubernetes/org.yaml:1392]: Valid point. We can discuss this later.

/lgtm cancel

**Comment by floreks** [comment]
lgtm

**Comment by shu-mutou** [comment]
Ref: https://github.com/kubernetes/community/pull/5840

**Pull request merged**